controllers/numaresourcesoperator_controller_test.go

```diff
@@ -261,7 +261,7 @@ var _ = Describe("Test NUMAResourcesOperator Reconcile", func() {
 
 			secondLoopResult, err := reconciler.Reconcile(context.TODO(), reconcile.Request{NamespacedName: key})
 			Expect(err).ToNot(HaveOccurred())
-			Expect(secondLoopResult).To(Equal(reconcile.Result{RequeueAfter: 5 * time.Second}))
+			Expect(secondLoopResult).To(Equal(reconcile.Result{RequeueAfter: 0}))
```

Comment:
let's add comments explaining what we expect and why (here and in the following code)

Reply:
Worth it indeed, added in all relevant places. Thanks.